spyder/plugins/plots/widgets/figurebrowser.py

```diff
@@ -145,59 +155,70 @@ def setup(self, mute_inline_plotting=None, show_plot_outline=None,
+        self.savefig_btn.shortcut_data = ('plots', 'save')
+        self.savefig_btn.installEventFilter(self)
 
         saveall_btn = create_toolbutton(
-                self, icon=ima.icon('save_all'),
-                tip=_("Save All Images..."),
-                triggered=self.save_all_figures)
+            self, icon=ima.icon('save_all'),
+            tip=_("Save all plots..."),
+            triggered=self.save_all_figures)
+        saveall_btn.shortcut_data = ('plots', 'save all')
+        saveall_btn.installEventFilter(self)
 
         copyfig_btn = create_toolbutton(
             self, icon=ima.icon('editcopy'),
-            tip=(_("Copy plot to clipboard as image (%s)") %
-                 CONF.get_shortcut('plots', 'copy')),
+            tip=_("Copy plot to clipboard as image"),
             triggered=self.copy_figure)
+        copyfig_btn.shortcut_data = ('plots', 'copy')
+        copyfig_btn.installEventFilter(self)
 
-        closefig_btn = create_toolbutton(
-                self, icon=ima.icon('editclear'),
-                tip=_("Remove image"),
-                triggered=self.close_figure)
+        self.closefig_btn = create_toolbutton(
+            self, icon=ima.icon('editclear'),
+            tip=_("Remove plot"),
+            triggered=self.close_figure)
+        self.closefig_btn.shortcut_data = ('plots', 'close')
+        self.closefig_btn.installEventFilter(self)
 
         closeall_btn = create_toolbutton(
-                self, icon=ima.icon('filecloseall'),
-                tip=_("Remove all images from the explorer"),
-                triggered=self.close_all_figures)
+            self, icon=ima.icon('filecloseall'),
+            tip=_("Remove all plots"),
+            triggered=self.close_all_figures)
+        closeall_btn.shortcut_data = ('plots', 'close all')
+        closeall_btn.installEventFilter(self)
 
         vsep1 = QFrame()
         vsep1.setFrameStyle(53)
 
         goback_btn = create_toolbutton(
-                self, icon=ima.icon('ArrowBack'),
-                tip=_("Previous figure ({})").format(
-                      CONF.get_shortcut('plots', 'previous figure')),
-                triggered=self.go_previous_thumbnail)
+            self, icon=ima.icon('ArrowBack'),
+            tip=_("Previous plot"),
+            triggered=self.go_previous_thumbnail)
+        goback_btn.shortcut_data = ('plots', 'previous figure')
+        goback_btn.installEventFilter(self)
 
         gonext_btn = create_toolbutton(
-                self, icon=ima.icon('ArrowForward'),
-                tip=_("Next figure ({})").format(
-                      CONF.get_shortcut('plots', 'next figure')),
-                triggered=self.go_next_thumbnail)
+            self, icon=ima.icon('ArrowForward'),
+            tip=_("Next plot"),
+            triggered=self.go_next_thumbnail)
+        gonext_btn.shortcut_data = ('plots', 'next figure')
+        gonext_btn.installEventFilter(self)
 
         vsep2 = QFrame()
```

Comment:
I know this is not related to this PR, but we shoul probably use longer descriptive names, we do not get points for shorter variables :-p